content/doc/developer/publishing/requesting-hosting.adoc

```diff
@@ -62,12 +63,42 @@ Alternately, you can configure your plugin to be released automatically whenever
 In this case you only need write permission to GitHub; you never need local credentials.
 Follow link:/redirect/continuous-delivery-of-plugins[this guide] for details.
 
-== Categorize the Plugin
+=== Automate your GitHub repository
+
+==== Activate GitHub actions
```

Comment:
Based on [this page](https://github.com/features/actions), it appears that GitHub Actions is the name of a product (i.e., a proper noun) and should therefore be capitalized.